crates/sui-indexer/src/models/epoch.rs

```diff
@@ -117,50 +117,107 @@ pub struct QueryableEpochSystemState {
     pub system_state: Vec<u8>,
 }
 
-impl StartOfEpochUpdate {
+#[derive(Default)]
+pub struct EpochStartInfo {
```

Comment:
derive `Default` here as it's a legit info for genesis epoch and it's used in checkpoint_handler.